charts/alertmanager/templates/statefulset.yaml

```diff
@@ -62,6 +62,10 @@ spec:
             - --webhook-url=http://127.0.0.1:{{ .Values.service.port }}/-/reload
           resources:
             {{- toYaml .Values.configmapReload.resources | nindent 12 }}
+          {{- if .Values.configmapReload.containerPort }}
+          ports:
+            - containerPort: {{ .Values.configmapReload.containerPort }}
+          {{- end }}
```

Comment:
```suggestion
          ports:
            - containerPort: 9533
```

Would this be bad?

Reply:
I didn't want my change to affect every user of the chart, that's why I kept it optional.

Reply:
True, I appreciate that, but now I am thinking what if we did, what would be the worst case scenario?

Reply:
People might already use the `9533` port number?

Reply:
Unlikely enough that I say let's just hardcode da thing... @monotek thoughts?

Reply:
Imho it does not hurt to have it configurable.

Reply:
very well